app/models/api/v3/context.rb

```diff
@@ -71,6 +71,30 @@ def exporter_context_node_type
         end
       end
 
+      def biome_nodes
+        biome_context_node_type = context_node_types.
+          joins(:node_type).
+          where('node_types.name' => NodeTypeName::BIOME).
```

Comment:
if there's only one we could use `find_by`

Reply:
👍